nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/test/java/org/nuxeo/ecm/automation/core/operations/users/TestSuggestUserEntries.java

```diff
@@ -276,4 +276,75 @@ public void testMixedEntries() throws Exception {
         }
     }
 
+    @Test
+    public void testSubGroupEntries() throws Exception {
+
+        // create top level group 'employees'
+        DocumentModel Employees = userManager.getBareGroupModel();
+        Employees.setPropertyValue("group:groupname", "Employees");
```

Comment:
`groupname` is an internal id, best to use lowercase: `employees`, same for all the other groups/subgroups below